etnawrapper/etna.py

```diff
@@ -275,11 +276,25 @@ def open_ticket(self, title: str, message: str, tags: List[str] = None, users: L
         content['tags'] = tags
         content['users'] = users
 
-        url = TICKET_URL
+        url = TICKETS_URL
         result = self._query(url, method='OPTIONS', raw=True)
         result = self._query(url, method='POST', data=content)
 
         return result
 
+    def close_ticket(self, ticket_id: int):
+        """Close a ticket."""
+        url = TICKET_URL.format(task_id=ticket_id)
+        result = self._query(url, method='DELETE')
+        return result
+
+    def get_tickets(self, ticket_id: int = None):
+        """Fetch the list of tickets."""
+        url = TICKETS_URL
+        if ticket_id is not None:
+            url = TICKET_URL.format(task_id=ticket_id)
+        result = self._query(url)
+        return result
```

Comment:
```suggestion
    def get_tickets(self):
        """Fetch the list of tickets."""
        url = TICKETS_URL
        result = self._query(url)
        return result
    
    
    def get_ticket(self, ticket_id: int):
        """Fetch the ticket matching `ticket_id`."""
        url = TICKET_URL.format(task_id=ticket_id)
        result = self._query(url)
        return result
```
Ca me semble plus explicite, si ca te convient :)